packages/AlertDialog/index.tsx

```diff
@@ -28,122 +29,109 @@ const ModalButton = styled.TouchableOpacity`
   align-items: center;
 `;
 
-const ModalInput = styled.TextInput<ColorProps>`
+const ModalInput = styled.TextInput`
   height: 40px;
   margin-top: 20px;
   padding: 4px;
   font-size: 14px;
-  border: 1px solid ${(props) => props.color};
+  border: 1px solid black;
 `;
 
-const ButtonText = styled.Text<ColorProps>`
+const ButtonText = styled.Text`
   font-size: 14px;
   font-weight: bold;
-  color: ${(props) => props.color};
+  color: black;
 `;
 
 type Styles = {
-  modalContainer?: StyleProp<ViewStyle>;
+  modal?: ModalProps['style'];
```

Comment:
Since `modal` is a parent component, I think this should go to `style` instead of `styles`.

Also in `styles`, I think `buttonContainer`, and `container`, `modalInput` can be included to customize styles.

Reply:
I strongly agree with @hyochan!